src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TFVarFileList.tsx

```diff
@@ -34,18 +34,28 @@ import { TFRemoteWizard } from './TFRemoteWizard'
 import { TFVarStore } from './TFVarStore'
 
 import InlineVarFile from './InlineVarFile'
-
+import { TFArtifactoryForm } from './TerraformArtifactoryForm'
 import css from './TerraformVarfile.module.scss'
 import stepCss from '@pipeline/components/PipelineSteps/Steps/Steps.module.scss'
 
 interface TfVarFileProps {
   formik: FormikProps<TerraformData>
   isReadonly?: boolean
   allowableTypes: MultiTypeInputType[]
+  getNewConnectorSteps?: any
+  setSelectedConnector?: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232248)
```suggestion
  setSelectedConnector?: unknown
```